knr1050Sup/knr1050.db

```diff
@@ -197,22 +156,22 @@ record(ai, "$(P)GET_STATUS") {
 	info(alarm, "Knr1050")
 }
 
-# /// For storing the parsed values of get_status
 record(calc, "$(P)BUFFER") {
+	field(DESC, "Parsed values of get_status")
 	field(CALC, "0")
 }
 
 record(ai, "$(P)TIMESTAMP") {
-	field(DESC, "Device timestamp in ms")
+	field(DESC, "Device time stamp in ms")
 	field(EGU, "ms")
 	
 	field(INP, "$(P)BUFFER.A CP")
 	
 	info(INTEREST, "HIGH")
 }
 
-record(mbbi, "$(P)INST_STATE") {
-    field(DESC, "The current instrument state.")
+record(mbbi, "$(P)DEV_STATE") {
```

Comment:
The is PV should probably just be called `STATUS:DEVICE`. It would be good to group all of the PVs that get their data from buffer be called `STATUS:%s` where `%s` is a string which indicates what `STATUS` they encode.